src/ui/server/helpers/summary-lists/your-business/sic-code-mapping/index.ts

```diff
@@ -0,0 +1,21 @@
+import { generateSicCodesValue } from '../../company-house-summary-list';
+import { DEFAULT } from '../../../../content-strings';
+import { ApplicationExporterSicCodes } from '../../../../../types';
+
+/**
+ * sicCodeMapping
+ * maps application sic code response to generate a string of the sic code values
+ * @param {Array<ApplicationExporterSicCodes>} applicationSicCodes
+ * @returns {String} sicCodes
+ */
+const sicCodeMapping = (applicationSicCodes?: Array<ApplicationExporterSicCodes>) => {
```

Comment:
How about `mapSicCodes`? Same with the file name

Reply:
we already have a mapSicCodes for adding to the db